### PR TITLE
Looks_like for vanilla monsters

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
```SUMMARY: Infrastructure "Add looks_like to vanilla monsters"```

#### Purpose of change
Work for #34230.

#### Describe the solution
Adds looks_like to all monsters in vanilla. Tileset whitelist is as follows(Whitelist based on the commonly seen variant of that category):
Birds: Chicken
UGVs: M16A4 UGV
Drones: Manhack
Cockroach & locust: Giant cockroach
Wasps & Bees: Giant bee
Non-wasp/bee flying insects: Giant fly
Ants: Giant ant
Spiders: Giant cellar spider
Fish: Trout for non-hostile, zhark for hostile
Equines: Horse
Rodents: Black rat(Doesn't include giant naked mole rat)
Felines: Cat
Large furred mammals: Black bear
Canines(includes zombie): Wolf
Milkable animals(sheep/cow): Cow
Deer: Moose
Swine: Pig
Mi-go: Mi-go(duh)
Mechsuits: Recon mechsuit
Reptiles: Rattlesnake
Triffids: Triffid(duh),
Turrets: M249 autonomous CROWS II
Zombie(mundane): Zombie
Zombie(Acidic): Acidic zombie
Zombie(Skeletal): Skeleton
Zombie(Shocker): Shocker zombie
Zombie(child): Zombie child
Zombie(Military): Zombie soldier
Zombie(feral): Feral hunter
Zombie(cult): Flesh angel
Zombie(Bloaters): Boomer
Zombie(large and not covered by others): Zombie hulk
Unsorted: Hound of Tindalos, Shia LaBeouf, Shoggoth, Thriller
#### Describe alternatives you've considered
TBD

#### Testing
To be done.

#### Additional context
None.

### CODE REVIEW

**Comment by barsoosayque** [comment]
If `copy-from` counts as `looks_like`, isn't it better to set `looks_like` for generic fish types (*mon_fish_tiny*, *mon_fish_medium*, etc) instead of the actual fish entries ?

File: data/json/monsters/reptile_amphibian.json [hunk 1]
```diff
@@ -113,6 +116,7 @@
   },
   {
     "id": "mon_rattlesnake_giant",
+    "looks_like": "mon_gator",
```
Comment (kevingranade) [inline data/json/monsters/reptile_amphibian.json]:
```suggestion
    "looks_like": "mon_rattlesnake",
```
Probably?

File: data/json/monsters/fish.json [hunk 2]
```diff
@@ -455,6 +468,7 @@
   },
   {
     "id": "mon_fish_lobster",
+    "looks_like": "mon_fish_trout",
```
Comment (kevingranade) [inline data/json/monsters/fish.json]: I think lobster and crawfish are different enough from fish to merit their own tile.
Probably make one look_like the other, no opinion which one.

File: data/json/monsters/fish.json [hunk 3]
```diff
@@ -484,6 +498,7 @@
   },
   {
     "id": "mon_fish_crayfish",
+    "looks_like": "mon_fish_trout",
```
Comment (kevingranade) [inline data/json/monsters/fish.json]: I think lobster and crawfish are different enough from fish to merit their own tile.
Probably make one look_like the other, no opinion which one.

File: data/json/monsters/reptile_amphibian.json (hunk 1 above)
Comment (LaVeyanFiend) [reply data/json/monsters/reptile_amphibian.json]: Definitely ;P

File: data/json/monsters/bird.json [hunk 4]
```diff
@@ -163,6 +167,7 @@
   },
   {
     "id": "mon_cockatrice",
+    "looks_like": "mon_chicken",
```
Comment (ZhilkinSerg) [inline data/json/monsters/bird.json]:
```suggestion
```

Comment (ZhilkinSerg) [reply data/json/monsters/bird.json]: It is defined already on line 185.

Comment (LaVeyanFiend) [reply data/json/monsters/bird.json]: To keep the formatting consistent, I shall remove the line 185 ``looks_like``.

File: data/json/monsters/bird.json [hunk 5]
```diff
@@ -194,6 +198,7 @@
   },
   {
     "id": "mon_chicken_chick",
+    "looks_like": "mon_chicken",
```
Comment (ZhilkinSerg) [inline data/json/monsters/bird.json]: There is the same string on line 216 already.

```suggestion
```

**Comment by I-am-Erk** [comment]
I am still really unsure that monsters should have the same looks-like treatment as other things. Having a skeletal juggernaut show up as a regular skeleton is not desirable behaviour.

File: data/json/monsters/bird.json (hunk 5 above)
Comment (LaVeyanFiend) [reply data/json/monsters/bird.json]: Removed Line 216's for consistent formatting.
(Finals began this week, I'm not dead on this yet 😛 )

**Comment by LaVeyanFiend** [comment]
> I am still really unsure that monsters should have the same looks-like treatment as other things. Having a skeletal juggernaut show up as a regular skeleton is not desirable behaviour.

Should I split it up more then, i.e. base skeleton and then the brute/juggernaut?

**Comment by I-am-Erk** [comment]
yeah as a general rule I would say not to have anything looks_like something of generally lower or higher threat

**Comment by LaVeyanFiend** [comment]
I'm just going to strip out the Mi-Go, zombies, and bosses in this PR, and do another one once an appropriate whitelisted option for those is figured out with respect to Erk's advice on being tricked on the danger of each mob.

**Comment by LaVeyanFiend** [comment]
I have no way to distinquishing what is OK or not to looks_like, so I'm closing this.

**Pull request closed without merging**